src/modules/auth/services/auth-service/auth.service.ts

```diff
@@ -23,7 +23,12 @@ class AuthService {
 	}
 
 	public redirectToLogin(query: StringifiableRecord) {
-		const returnToUrl = stringifyUrl({ url: config.public.origin, query });
+		// eslint-disable-next-line @typescript-eslint/no-unused-vars
+		const { showAuth, ...queryParams } = query;
```

Comment:
Eventueel kunnen we in de `AuthModal` waar er geredirect wordt naar de login, eerst de `showAuth` query op false zetten?

Reply:
https://github.com/viaacode/hetarchief-client/blob/master/src/pages/index.tsx#L140

```
			<AuthModal isOpen={showAuthModal && !user} onClose={onCloseAuthModal} />
```

Reply:
good points

I've fixed both suggestions.

`{showAuthModal && !user}` hides the modal, but then the url still contains `http://localhost:3200/?showAuth=1` which is a bit confusing